services/workflows-service/src/providers/file/utils/stream-to-base64.ts

```diff
@@ -0,0 +1,13 @@
+import { Base64Encode } from 'base64-stream';
+import concat from 'concat-stream';
+
+export const streamToBase64 = (stream: NodeJS.ReadableStream): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    const base64Encoder = new Base64Encode();
+
+    stream
+      .pipe(base64Encoder)
+      .pipe(concat(base64 => resolve(base64 as unknown as string)))
```

Comment:
Why is a type assertion needed here?

Reply:
P3

Reply:
Type is casted wrongly out of `concat`, returns `Buffer` in fact its string.